idaes/models/properties/modular_properties/reactions/equilibrium_forms.py

```diff
@@ -131,6 +131,20 @@ def build_parameters(rblock, config):
             mutable=True, initialize=1e-4, doc="Smoothing parameter for smooth maximum"
         )
 
+        rblock.s_norm = Param(
+            mutable=True,
+            initialize=1e-4,
+            doc="Normalizing factor for solid precipitation term",
+        )
+        if hasattr(rblock, "k_eq_ref"):
+            rblock.s_norm.value = rblock.k_eq_ref.value
```

Comment:
I think it would be better to do this as `rblock.s_norm.set_value(rblock.k_eq_ref)`; this will let the set value method handle any necessary unit conversions or edge-cases that might appear.

Reply:
`rblock.k_eq_ref` is not a Param but a Var, it does not allow me to use `set_value` here.

Reply:
@andrewlee94 
![image](https://github.com/IDAES/idaes-pse/assets/58192375/b8b83eed-3a2e-42b2-8cef-ee5ccf2e47ff)

Reply:
I thought that should have worked - either way as you noted it works if you use `value()` so we should go with that.